charts/dawarich/values.yaml

```diff
@@ -104,6 +104,43 @@ photon:
 #    limits:
 #      memory: 512Mi
 
+envoyProxy:
```

Comment:
This should probably be under the existing Photon configuration and should be called something like loadbalancerProxy